param-docs/parameter-pages/MQTT/DomoticzTopicIn.md

```diff
@@ -0,0 +1,6 @@
+# Parameter `DomoticzTopicIn`
+Default Value: `domoticz/in`
+
+MQTT IN topic for Domoticz integration, under which the counters are published.
+
+Counter Idx is required.
```

Comment:
does this refer to `Parameter <NUMBER>.DomoticzIDX`? If so, please name it that ways

Reply:
It does. I have updated the parameter name as requested. Thanks